.gitignore

```diff
@@ -13,3 +13,4 @@ npm-debug.log
 coverage
 
 .DS_Store
+.idea
```

Comment:
@jcheroske this is unrelated, sometimes PRs are not merged because of things like this.